Podfile

```diff
@@ -29,7 +29,8 @@ target 'WordPress', :exclusive => true do
   pod 'RxSwift', '~> 2.1.0'
   pod 'SVProgressHUD', '~>1.1.3'
   pod 'UIDeviceIdentifier', '~> 0.1'
-
+  pod 'Fabric'
```

Comment:
I'm going to suggest we not include the `Fabric` pod as its a dependency of `Crashlytics` anyway.